docs/core/whats-new/dotnet-9/libraries.md

```diff
@@ -81,7 +80,13 @@ For more information, see the [API proposal](https://github.com/dotnet/runtime/i
 
 ## Cryptography
 
-For cryptography, .NET 9 adds a new one-shot hash method on the <xref:System.Security.Cryptography.CryptographicOperations> type. It also adds new classes that use the KMAC algorithm.
+- [CryptographicOperations.HashData() method](#cryptographicoperationshashdata-method)
```

Comment:
FYI: I wrote a tool to generate these TOC lists if that is useful to you.

Reply:
Thanks - we also have a helper function in the [learn-markdown extension](https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-markdown) we use.

Reply:
FYI: @jongalloway @jamesmontemagno